tensorboard/webapp/widgets/data_table/data_table_component.ng.html

```diff
@@ -104,7 +56,7 @@
       <button
         mat-icon-button
         class="add-column-btn"
-        (click)="openColumnSelector($event)"
+        (click)="openColumnSelector({event: $event})"
```

Comment:
To allow ContextMenuComponent to propagate openColumnSelector parameters through its output, we need to make openColumnSelector's parameters an object as an Event can only be an object.